projects/storefrontlib/cms-structure/page/page-layout/page-layout.service.ts

```diff
@@ -130,9 +140,9 @@ export class PageLayoutService implements OnDestroy {
     configAttribute: string,
     section?: string,
     breakpoint?: BREAKPOINT
-  ): SlotConfig {
+  ): SlotConfig | undefined {
```

Comment:
Why undefined instead of null?

Reply:
`undefined` is a required type. It can also be `SlotConfig | undefined | null`